src/components/Entry.vue

```diff
@@ -53,9 +60,9 @@
     <span v-if="isInvalidInputMessageShown" class="mx-auto text-red-700 text-base mb-1">{{
       invalidInputMessage
     }}</span>
-    <span v-if="isInvalidLoginMessageShown" class="mx-auto text-red-700 text-base mb-1">{{
+    <!-- <span v-if="isInvalidLoginMessageShown" class="mx-auto text-red-700 text-base mb-1">{{
       invalidLoginMessage
-    }}</span>
+    }}</span> -->
```

Comment:
is this needed? if not, can you remove?

Reply:
So actually made a change in the `signup` PR.